package.json

```diff
@@ -92,7 +92,7 @@
         "cross-env": "^7.0.3",
         "deep-equal": "^2.0.5",
         "eslint": "^8.57.1",
-        "eslint-config-prettier": "^9.1.0",
+        "eslint-config-prettier": "^10.0.0",
```

Comment:
@vladfrangu any idea why we have this here? I thought we don't use prettier in this repo anywhere.

Reply:
Iirc I just used this as a quick "disable all formatting rules from eslint because they're handled by biome"

Reply:
Ok sounds safe to merge then